hydroflow/src/scheduled/net/network_vertex.rs

```diff
@@ -11,17 +12,26 @@ use crate::scheduled::{
     handoff::VecHandoff,
 };
 
-use super::Message;
-
 pub type Address = String;
 
 // These methods can't be wrapped up in a trait because async methods are not
 // allowed in traits (yet).
 
 impl Hydroflow {
-    // Listen on a port and send any messages received along the output edge.
-    // Returns the port bound to.
-    pub async fn inbound_tcp_vertex(&mut self) -> (u16, OutputPort<VecHandoff<Message>>) {
+    // TODO(justin): this needs to return a result/get rid of all the unwraps, I
+    // guess we need a HydroflowError?
+    /// Begins listening on some TCP port. Returns an [OutputPort] representing
+    /// the stream of messages received. Currently there is no notion of
+    /// identity to the connections received, if they are to be attached to some
+    /// participant in the system, that needs to be included in the message
+    /// directly.
+    ///
+    /// The messages will be interpreted to be bincode-encoded, length-delimited
+    /// messages, as produced by [outbound_tcp_vertex].
```

Comment:
Link needs to be `[Self::outbound_tcp_vertex]` or maybe `[Hydroflow::outbound_tcp_vertex]`